tests/generate_packageTest.py

```diff
@@ -0,0 +1,24 @@
+import os
+import unittest
+from bears.generate_package import touch, create_file_from_template
+
+
+class touchTest(unittest.TestCase):
+
+    def test_file_doesnt_exist(self):
+        touch('TestFile.py')
+        self.assertTrue(os.path.exists('TestFile.py'))
```

Comment:
I see that your test really only covers the case where it does not exist. Maybe try creating `test_file_exists`